conf/test.config

```diff
@@ -24,6 +24,7 @@ params {
     input = "https://raw.githubusercontent.com/nf-core/test-datasets/rnaseq/samplesheet/v3.10/samplesheet_test.csv"
 
     // Genome references
+    load_igenomes      = true
```

Comment:
not familiar with this params

Reply:
oh sorry, you rewrote the igenomes_ignore..

Reply:
The thing is, if we we strongly recommend against using something, and then have using that thing as the default behaviour, we're being very confusing for users IMO. 

I'm all for consistency, but sometimes we have to be the change that's needed ;-)

Reply:
We still have this exact params in differentialabundance

Reply:
> We still have this exact params in differentialabundance

Yes, it is important that that workflow allowed selection of references in the same way as this workflow since people so often use them together. If I can deprecate igenomes here, I will do the same there.

Reply:
We have a mentee in the mentorship programs that's currently working on piecing together rnaseq and rnavar.
Another one who's piecing together rnavar and sarek.
If we were to update such a low level params in a pipeline, we start a cascade that touches all.
I'd rather we tackle that via TEMPLATE update, or when we finally manage to ditch iGenomes for good (or update it).

Reply:
Oh alright, I'll roll back and add some additional documentation to mitigate the confusion.

Reply:
Better?